python/tank/descriptor/bundle_cache_usage/logger.py

```diff
@@ -0,0 +1,281 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import threading
+import Queue
+
+from ...import LogManager
+from .errors import BundleCacheUsageTimeoutError
+from .database import BundleCacheUsageDatabase as BundleCacheUsageDatabase
+from . import BundleCacheUsageMyLogger as tmp_log
+
+
+log = LogManager.get_logger(__name__)
+
+
+class BundleCacheUsageLogger(threading.Thread):
```

Comment:
i get constantly confused by `logger` - it sounds like we are logging messages. Could we use the verb tracker isntead of logger? I think that will help future readers of the code.